tests/test_io/good/condition_only_input.catala_en

```diff
@@ -19,10 +19,10 @@ $ catala Dcalc -s B
 let B =
   λ (B_in: B_in{}) →
     let a.x : bool = error_empty ⟨true ⊢ false⟩ in
-    let result : A_out{"y_out": integer} = A (A_in {"x_in"= a.x}) in
-    let a.y : integer = result."y_out" in
+    let result : A{"y": integer} = A (A_in {"x_in"= a.x}) in
```

Comment:
The dissymetry between `_in` as a suffix for the input and no suffix for the output is weird. But not sure what we can do about it.